### PR TITLE
use external protobuf msgs

### PR DESCRIPTION
**Pull request by TarikViehmann**
Use a submodule for protobuf messages for easier management.

### CODE REVIEW

**Comment by techtasie** [comment]
Maybe add a check if the submodule is there in the cmake...

**Comment by TarikViehmann** [comment]
good idea, done

File: cmake/git.cmake [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# *****************************************************************************
+cmake_minimum_required(VERSION 3.0)
+project(MyProject)
```
Comment (snoato) [inline cmake/git.cmake:23]: is this right?

**Review by snoato — APPROVED** [review]
some questions, approved if negligible

File: cmake/git.cmake (hunk 1 above)
Comment (TarikViehmann) [reply cmake/git.cmake]: you are correct

**Review by snoato — APPROVED** [review]
:D

**Pull request merged**